CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Example app additions:
+  - A row of 3 plain single selectors ([#19](https://github.com/rhventures/react-native-dropdown-selector/pull/19)).
+  - A row of 3 plain multi selectors ([#19](https://github.com/rhventures/react-native-dropdown-selector/pull/19)).
+  - A customized single selector ([#19](https://github.com/rhventures/react-native-dropdown-selector/pull/19)).
+  - A customized multi selector ([#19](https://github.com/rhventures/react-native-dropdown-selector/pull/19)).
```

Comment:
Each item should get its own line instead of being a subpoint. It will look a little repetitive, but that is fine.